### DISCUSSION TITLE
Actividad 0 - Pieroni, Nicolas Agustin

### DISCUSSION THREAD

**Discussion by Nic0-GitHub** [frlp-utn-ingsoft/catedra#36] [Show and tell]
https://github.com/Nic0-GitHub/django-ing-de-software

**Comment by RodrigoJacznik** [comment]
Buenas,
Los mensajes de los commits estan bien pero la app no funciona. Cuando quiero votar me tira un error. Hay un error en `DetailView`.
Por favor fixealo para aprobar la actividad.
Saludos

**Reply by Nic0-GitHub** [reply]
Buenas. Cuando cambié el nombre de las plantillas, olvidé actualizarlo en las vistas. He realizado la corrección y algunas mejoras en  las plantillas de la app de 'polls'.

**Reply by RodrigoJacznik** [reply]
Genial, quedo muy buena la app.
Los estilos quedaron muy bien.
El tp esta aprobado.
